internetarchive/cli/ia_upload.py

```diff
@@ -85,8 +86,23 @@
 )
 
 
+def _check_files(files):
+    return [
+        file
```

Comment:
This is very strangely formatted and should at least have `file for file in files` on a single line to make it more apparent what is happening.